### DISCUSSION TITLE
Usando Prefixo nas Rotas

### DISCUSSION THREAD

**Discussion by valdirdill** [HashLoad/horse#392] [General]
Bom dia,

Estou querendo usar um prefixo nas rotas. 
Testei assim _THorse.Group.Prefix'meuPrefixo').Patch('/login', doLogin)_ e funciona beleza.

A dúvida é: tem como definir esse prefixo de forma global, para todas as rotas? Ou realmente precisa informar ele em cada rota?

Obrigado!

**Comment by viniciussanchez** [comment]
Você pode definir de forma global assim:
`THorse.Routes.Prefix('meuPrefixo');`
